content/working-with-azure-key-vault-in-csharp/index.md

```diff
@@ -117,11 +117,11 @@ To enable the system-assigned managed identity for the Azure Web App we just cre
 
 Figure 3
 
-## **Create and Configure Azure Key Vault**
+## Create and Configure Azure Key Vault
 
 In this section we’ll examine how we can create and configure an Azure key Vault instance.
```

Comment:
Changes: In this section`,` we’ll examine how `to` create and configure an Azure key Vault instance.